assets/src/components/AssignmentTable.js

```diff
@@ -137,7 +137,7 @@ function AssignmentTable (props) {
   const [previousWeek, setPreviousWeek] = useState()
 
   const weeksPresent = useRef([])
-  const shouldScrollToCurrentWeek = useRef(false)
+  const shouldScrollToCurrentWeek = useRef(true)
```

Comment:
Given the name I thought this made more sense to use this way.
Not directly related to this functionality.